parent/pom.xml

```diff
@@ -144,6 +144,42 @@
     <version.jna-platform>5.14.0</version.jna-platform>
     <version.msal4j>1.15.0</version.msal4j>
 
+    <version.commons-io>2.16.0</version.commons-io>
+    <version.thymeleaf>3.1.2.RELEASE</version.thymeleaf>
+    <version.unboundid-ldapsdk>6.0.11</version.unboundid-ldapsdk>
+    <version.parsson>1.1.5</version.parsson>
+    <version.springdoc>2.2.0</version.springdoc>
+    <version.jakarta.json>2.0.1</version.jakarta.json>
```

Comment:
can we use 2.1.3 as `version.jakarta.json-api` ?

Reply:
I would expect renovate will handle this

Reply:
this is used by `org.glassfish:jakarta.json`. 2.0.1 seems to be the last version , it seems it came with `jakarta.json-api` 2.0.1 , I hope it is compatible with 2.1.3 set by the parent